### PR TITLE
Division of training loss by no. of batches

### PR DESCRIPTION
**Pull request by ahmedhshahin**
Hello,

The original version of the code divides the accumulated training loss by the number of images in the dataset `len(train_loader.dataset)` which - I believe - should not be accurate. The correct implementation should divide by the number of batches `len(train_loader)`, as the returned loss from `CoxLoss` is already averaged across each batch.

Thanks

### CODE REVIEW

**Comment by Richarizardd** [comment]
Thank you for the fix into this issue! :)
